features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_WORKFLOW.siddhi

```diff
@@ -0,0 +1,29 @@
+/*
+    Copyright (c) 2018, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+@App:name("APIM_WORKFLOW")
+@App:description("Workflow Stream Definition")
+
+@source(type = 'wso2event', wso2.stream.id = 'org.wso2.apimgt.statistics.workflow:1.0.0', 
+	@map(type = 'wso2event'))
+define stream workflowStream(
```

Comment:
Is this contain only the stream definition?

Reply:
yes

Reply:
what's the purpose?

Reply:
it was kind of added if someone wants to create custom implementations related to workflows. APIM sends events for this stream (in previous versions also it was done).